src/test/validatorTest.ts

```diff
@@ -1146,6 +1147,7 @@ describe('validator', () => {
 
         it('should be able to determine the type of every property of every propertytype', () => {
             for (let propertyTypeName in awsResources.PropertyTypes) {
+                if (propertyTypeName == 'Tag') { continue; }
```

Comment:
?

Reply:
This test skips checking the `Tag` property type as it's no longer needed nor do I believe it makes sense anymore as the correct reference should be `ResourceType.Tag` and those are being checked.